### PR TITLE
Display facilities by district - Analytics Cache Jobs

### PR DESCRIPTION
**Pull request by timmyjose**
Story: https://www.pivotaltracker.com/n/projects/2184102/stories/165465958
Scope: Analytics Cache jobs for Districts and separate Sidekiq jobs for each facility
Depends on: https://github.com/simpledotorg/simple-server/pull/390

### CODE REVIEW

File: app/controllers/analytics/districts_controller.rb [hunk 1]
```diff
@@ -30,16 +31,20 @@ def set_organization
 
   def set_facilities
     facilities_by_district = policy_scope(@organization.facilities).group_by(&:district).sort.to_h
-    @facilities = facilities_by_district[@district.id].sort_by(&:name)
-    @district.facilities_ids = @facilities&.map(&:id)
+    @district.facilities = facilities_by_district[@district.id].sort_by(&:name)
```
Comment (govindkrjoshi) [inline app/controllers/analytics/districts_controller.rb]:
```suggestion
    @district.facilities = policy_scope(@organization.facilities).where(district: @district.id)
```

Comment (timmyjose) [reply app/controllers/analytics/districts_controller.rb]: Fixed.

File: app/jobs/concerns/cacheable.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+module Cacheable
```
Comment (kitallis) [inline app/jobs/concerns/cacheable.rb]: We can call this `AnalyticsCacheable`

Comment (timmyjose) [reply app/jobs/concerns/cacheable.rb]: Updated.

**Pull request merged**